src/modules/hip/kernel/histogram.cpp

```diff
@@ -39,9 +39,9 @@ void partial_histogram_pln( unsigned char *input,
         unsigned char pixelR = input[pixId];
         unsigned char pixelG = input[pixId + width * height];
         unsigned char pixelB = input[pixId + 2 * width * height];
-        atomicInc(&tmp_histogram[pixelR], 1);
-        atomicInc(&tmp_histogram[pixelG], 1);
-        atomicInc(&tmp_histogram[pixelB], 1);
+        atomicAdd(&tmp_histogram[pixelR], 1);
```

Comment:
Is atomicAdd good option. I thought we can't use any atomics

Reply:
@rrawther Ubuntu build seems to pass with atomicAdd. Ideally, both should work as they did on openCL/cuda/hip, but for some reason only atomicInc() fails on Rocm6.0.0 HIP